src/golang/lib/engine/utils_test.go

```diff
@@ -0,0 +1,41 @@
+package engine
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/aqueducthq/aqueduct/lib/collections/shared"
+	"github.com/stretchr/testify/require"
+)
+
+func TestExtractAwsCredentials(t *testing.T) {
+	credentialsFilepath := filepath.Join(t.TempDir(), "credentials_test")
+	f, err := os.Create(credentialsFilepath)
```

Comment:
nit: is this file deleted

Reply:
yep, the testing library removes all files and the temp directory when the test finishes.